compiler/noirc_evaluator/src/ssa/acir_gen/acir_ir/acir_variable.rs

```diff
@@ -492,7 +492,7 @@ impl<F: AcirField> AcirContext<F> {
             self.sub_var(sum, mul)
         } else {
             // Implement OR in terms of AND
-            // (NOT a) NAND (NOT b) => a OR b
+            // (NOT a) AND (NOT b) => a OR b
```

Comment:
```suggestion
            // (NOT a) AND (NOT b) => NOT (a OR b)
```
`!a && !b` would equal `!(a || b)`, not `a || b`. The original NAND is correct. I'm guessing we just missed a NOT around the result